Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM hathitrust/feed_base:buster
+
+RUN apt-get install -y libtest-class-perl libswitch-perl libtest-spec-perl libtest-mockobject-perl
+
+WORKDIR /htapps/babel/imgsrv
+COPY . /htapps/babel/imgsrv
+
+RUN git config --global url."https://github.com/".insteadOf git@github.com:
+RUN git config --global url."https://".insteadOf git://
+
+RUN git submodule init
+RUN git submodule update
+RUN ln -s imgsrv/vendor/common-lib/lib ../mdp-lib
+RUN ln -s /htapps/babel /htapps/test.babel
+RUN cd ./sql; ln -s ../vendor/common-lib/lib/sql/*.sql .
+
+RUN wget -O /tmp/netpbm-sf-10.73.32_amd64.deb https://sourceforge.net/projects/netpbm/files/super_stable/10.73.32/netpbm-sf-10.73.32_amd64.deb/download
```

Comment:
I'd recommend moving this wget and the symlinking above the `COPY .` and submodule setup, since as-is this will cause having to re-fetch netpbm and make the symlinks every time you update the app and build the image -- generally the Dockerfile should be structured as the least-likely to change things happening first and the most-likely things to change happening at the end

Reply:
This part _is_ annoying when I rebuild the container; I had figured the only solution was something like `hathitrust/base_base:buster` that extended `hathitrust/feed_base:buster` _once_. Will give this a try.